plugins/scaffolder-backend-module-github/src/actions/helpers.ts

```diff
@@ -304,6 +310,18 @@ export async function createGithubRepoWithCollaboratorsAndTopics(
     }
   }
 
+  if (oidcCustomization) {
+    await client.request(
+      'PUT /repos/{owner}/{repo}/actions/oidc/customization/sub',
```

Comment:
This is making a request directly as a `.rest.` method isn't available.

I traced this down to some broken filtering here https://github.com/gr2m/github-openapi-graphql-query/pull/131, but this probably takes some time until it bubbles through all the layers.

The types for this request interestingly exist, so regressions should be noticed quickly.